zip-0226.rst

```diff
@@ -325,6 +325,111 @@ A ZSA Asset Burn description is encoded in a transaction as an of a AssetBurn ty
+
+    txid_digest
+    ├── header_digest
+    ├── transparent_digest
+    │   ├── prevouts_digest
+    │   ├── sequence_digest
+    │   └── outputs_digest
+    ├── sapling_digest
+    │   ├── sapling_spends_digest
+    │   │   ├── sapling_spends_compact_digest
+    │   │   └── sapling_spends_noncompact_digest
+    │   ├── sapling_outputs_digest
+    │   │   ├── sapling_outputs_compact_digest
+    │   │   ├── sapling_outputs_memos_digest
+    │   │   └── sapling_outputs_noncompact_digest
+    │   └── valueBalance
+    └── orchard_digest
+    │   ├── orchard_actions_compact_digest
+    │   ├── orchard_actions_assetid_digest
+    │   ├── orchard_actions_memos_digest
+    │   ├── orchard_actions_noncompact_digest
+    │   ├── flagsOrchard
+    │   ├── valueBalanceOrchard
+    │   └── anchorOrchard
+    └── issuance_digest
+        ├── issue_actions_digest
+        └── issuanceValidatingKey
+
+In the specification below, nodes of the tree are presented in depth-first order.
+
+txid_digest
+-----------
+A BLAKE2b-256 hash of the following values ::
+
+   T.1: header_digest       (32-byte hash output)
+   T.2: transparent_digest  (32-byte hash output)
+   T.3: sapling_digest      (32-byte hash output)
+   T.4: orchard_digest      (32-byte hash output)
+   T.5: issuance_digest     (32-byte hash output)
+
+The personalization field remains the same as in ZIP 244 [#zip-0244]_. The branches ``T.1: header_digest``, ``T.2: transparent_digest``, and ``T.3: sapling_digest`` are as defined in ZIP 244.
+
+T.4: orchard_digest
+```````````````````
+When Orchard Actions are present in the transaction, this digest is a BLAKE2b-256 hash of the following values ::
+
+   T.4a: orchard_actions_compact_digest      (32-byte hash output)
+   T.4b: orchard_actions_assetid_digest      (32-byte hash output)
+   T.4c: orchard_actions_memos_digest        (32-byte hash output)
+   T.4d: orchard_actions_noncompact_digest   (32-byte hash output)
+   T.4e: flagsOrchard                        (1 byte)
+   T.4f: valueBalanceOrchard                 (64-bit signed little-endian)
+   T.4g: anchorOrchard                       (32 bytes)
+
+The value of ``T.4a: orchard_actions_compact_digest`` is the same as in ZIP 244 [#zip-0244]_. 
+
+T.4b: orchard_actions_assetid_digest
+''''''''''''''''''''''''''''''''''''
+
+A BLAKE2b-256 hash of the subset of Zcash Shielded Asset identifier data for all Orchard Actions belonging to the transaction. For each Action, the following elements are included in the hash::
+
+   T.4b.i: assetid (field encoding bytes)
+
+The personalization field of this hash is set to::
+
+  "ZTxIdOrcActAHash"
```

Comment:
Think we need to remove it as described above.

Reply:
Removed.